_posts/2018-04-16-so-you-want-to-optimize-grpc-part-2.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title: So You Want to Optimize gRPC - Part 2
+published: true
+permalink: blog/optimizing-grpc-part-2
+author: Carl Mastrangelo
+author-link: https://carlmastrangelo.com/
+company: Google
+company-link: https://www.google.com
+---
+
+How fast is gRPC?  Pretty fast if you understand how modern clients and servers are built.  In
+[part 1](/blog/optimizing-grpc-part-1), I showed how to get an easy **60%** improvement.  In this 
+post I show how to get a **10000%** improvement.
+
+<!--more-->
+
+## Setup
+
+As in [part 1](/blog/optimizing-grpc-part-1), we will be starting with an existing, Java based, 
+key-value service.  The service will be offering concurrent access for creating, reading, updating,
```

Comment:
The service will be offering -> the service will offer

Reply:
Done.